tests/testthat/test-boundsPredprob.R

```diff
@@ -0,0 +1,33 @@
+# boundsPostProb ----
```

Comment:
let's do this in the other PR https://github.com/Genentech/phase1b/pull/105 to not mix up

Reply:
oups, thought I was indeed on #105 when I did this, true that.